app/portainer/components/datatables/datatable-header/datatableHeader.html

```diff
@@ -0,0 +1,5 @@
+<a ng-click="$ctrl.changeOrderBy($ctrl.sortKey)">
```

Comment:
I'm not keen to introduce refactor outside of the module in any work that will be merged into the `edge-compute` branch. This will lead to potential conflicts later on with multiple teams working on the develop branch.

This refactor should target all the tables and not only a few of them otherwise we'll just add more complexity by breaking the consistency.

Open to discuss this though.

Reply:
I think this is the way we want to go as it removes code duplication. Most of the datatables are copy-pasted from each other (that what I feel). I had a few bugs with the headers before I decided to refactor it. I understand consistency, and I'm open to doing a refactor PR that will change the whole tables. 
I don't think this will introduce conflicts as it's only in this new datatable.

Reply:
Yup agree with the code duplication, it's just that this is out of the scope of the PR.

Reply:
:+1: